pinephone/vibrator/FFDevice.cpp

```diff
@@ -0,0 +1,149 @@
+#include "vibrator-impl/FFDevice.h"
```

Comment:
Please apply Google clang-format style for this project

Reply:
Done.